java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S3752_java.json

```diff
@@ -1,6 +1,6 @@
 {
-  "title": "\"@RequestMapping\" methods should specify HTTP method",
-  "type": "VULNERABILITY",
+  "title": "Allowing both safe and unsafe HTTP methods is security-sensitive",
```

Comment:
It seems that Eric is reworking this rule (SONARJAVA-3376), it requires some change in the rule, I believe we should not update the json and HTML now.